webextensions/api/contentscripts.json

```diff
@@ -0,0 +1,74 @@
+{
+  "webextensions": {
+    "api": {
+      "contentScripts": {
+        "RegisteredContentScript": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/contentScripts/RegisteredContentScript",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "59"
+              },
+              "firefox_android": {
+                "version_added": "59"
+              },
+              "opera": {
+                "version_added": false
+              }
+            }
+          },
+          "unregister": {
```

Comment:
Mmmh, not 100% sure, but I think register and unregister are 1 level too deep. @wbamberg, can you double check this?

Reply:
I don't think so: `register()` is a top level function in the namespace, but `unregister()` is a method of the `RegisteredContent Script object. So they are at different levels:

```
    "webextensions":
        "api"
            "contentScripts"
                "RegisteredContentScript"
                    "__compat"
                    "unregister"
                "register"
```

Running `$ npm run render webextensions.api.contentScripts 10 true > ~/Desktop/test.html 
` gives me the expected table:

<img width="726" alt="screen shot 2018-01-15 at 9 08 32 am" src="https://user-images.githubusercontent.com/432915/34954315-c94318d2-f9d4-11e7-9eb2-d90d773e2a7f.png">

Does that make sense, or am I missing something here?

But, this made me look again at this PR and I realised I got the case of the filename wrong, so I have corrected that.

Reply:
It does make sense. Thanks!